src/data_management/rasters/RasterForm.js

```diff
@@ -961,10 +1081,26 @@ class RasterFormModel extends Component {
                   </div>
                 ) : (
                   <div className={inputStyles.InputContainer}>
-                    <FormattedMessage
-                      id="rasters.please complete the form before submitting"
-                      defaultMessage="Please complete the form for submitting"
-                    />
+                    <button
+                      type="button"
+                      className={`${buttonStyles.Button} ${buttonStyles.Invalid}`}
+                      style={{
```

Comment:
Hi Lirry ziet er goed uit.
Nog een paar kleine puntjes.
De classname: buttonStyles.Invalid lijkt niet te werken. In de html komt deze als 'undefined'